client/flutter/lib/pages/protect_yourself.dart

```diff
@@ -180,7 +190,7 @@ class ProtectYourselfCard extends StatelessWidget {
             Padding(
               padding: const EdgeInsets.symmetric(
                 horizontal: 15,
-                vertical: 20,
+                vertical: 22,
```

Comment:
I'm not sure that this is what @hspinks specified as `Line height 22pt`. This is padding, while line height sounds more like something for `TextStyle`. But I don't see that being used anywhere in the app 🤔 ..
More info in TextStyle line height in any case: https://api.flutter.dev/flutter/painting/TextStyle/height.html

Reply:
@PieterAelse I agree with you, but if you add 22 inside the top, the components are deformed, so choose to look at the design that comes in figma and find this solution. 
this is an example using height 22.
![imagen](https://user-images.githubusercontent.com/59377864/78191475-fb06b700-746d-11ea-8169-b964cb0b6ac8.png)

Reply:
@PieterAelse Estoy de acuerdo con usted, pero si agrega 22 de altura, los componentes se deforman, por lo que decidí mirar el diseño que viene en figma y encontré esta solución.
Este es un ejemplo de altura 22.
![imagen](https://user-images.githubusercontent.com/59377864/78191586-3d2ff880-746e-11ea-85b5-7d479c153478.png)

Reply:
@PieterAelse I agree with you, but if you add 22 height, the components are deformed, so i choosed to look at the design that comes in figma and found this solution.
this is an example using height 22.
![imagen](https://user-images.githubusercontent.com/59377864/78191627-4faa3200-746e-11ea-8ce2-bb572279c50e.png)

Reply:
Sorry for all the comments I was confused about pull.

Reply:
As the linked documentation describes, the line height is used "as a multiple of the font size". So in order to achieve line height 22 it would be [current font size] x [find X] = 22 😛 

But like I said, I'm not seeing any line height usage in `TextStyle` in other places in the app. So I'm not sure, maybe it's already the default line height of the font we use? Because we're also not changing font size here.

I'm fine with the 2 extra px of padding, but I'm not sure if that ticks the box on the issue.

Reply:
Ok, i have no problem changing it, i will fix it and do it using height as it tells me. I'm on it.

Reply:
@PieterAelse Exact changes are already made